### PR TITLE
# TASK 22999* Add point sign in-out and Cloud Cache

### PR DESCRIPTION
**Pull request by WillyMoselhy**
Added a note regarding the impact of Cloud Cache during sign-in / out. and link to review local storage recommendations.

### CODE REVIEW

**Comment by cmcclister** [comment]
Learn Build status updates of commit _[45df479](https://github.com/WillyMoselhy/cloud-adoption-framework/commits/45df479a0d33c28420520073e989388ba091775a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md](https://github.com/WillyMoselhy/cloud-adoption-framework/blob/patch-1/docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/0286ad65-6ec8-0c70-0cae-2edc1504c701/202212191729378909-1031/BuildReport?accessString=95021f760a2829c7f83247532aaaf2ab07f16dec0a482231923d139460fc4b5c).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0286ad65-6ec8-0c70-0cae-2edc1504c701?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by ShannonLeavitt** [comment]
@igorpag 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by igorpag — APPROVED** [review]

**Comment by igorpag** [comment]
#sign-off

File: docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ For an Azure Virtual Desktop host pool, you can use either an *active-active* or
 - For virtual machines (VMs) in each region, invert the Cloud Cache registry entry that specifies locations to give precedence to the local cache registry.
 - Load balancing incoming user connections can't take proximity into account. All hosts are equal, and users might be directed to a remote (and not optimal) Azure Virtual Desktop host pool VM.
 - This configuration is limited to a *pooled* (shared) host pool type. For a *personal* (dedicated) type, when a desktop is assigned to a user on a certain session host VM, the desktop doesn't change, even if the VM isn't available.
+- Cloud Cache doesn't improve the users' sign-on and sign out experience when using poor performing storage. It's common for environments using Cloud Cache to have slightly slower sign-on and sign out times, relative to using traditional VHDLocations, using the same storage. [Review the FSLogix Cloud Cache documenation for recommendation regarding local cache storage.](/fslogix/cloud-cache-resiliency-availability-cncpt)
```
Comment (ShannonLeavitt) [inline docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md:42]:
```suggestion
- Cloud Cache doesn't improve the users' sign-on and sign out experience when using poor performing storage. It's common for environments using Cloud Cache to have slightly slower sign-on and sign out times, relative to using traditional VHDLocations, using the same storage. [Review the FSLogix Cloud Cache documentation for recommendations regarding local cache storage](/fslogix/cloud-cache-resiliency-availability-cncpt).
```
Typo & minor edits

**Comment by ShannonLeavitt** [comment]
@WillyMoselhy @igorpag - The link in line 42 doesn't seem to be working. Can you fix it?

#hold-off

**Review by igorpag — APPROVED** [review]

**Comment by igorpag** [comment]
#sign-off

**Comment by ShannonLeavitt** [comment]
@WillyMoselhy @igorpag - Please provide a working link for line 42. If you can add the link here in the comments, I can commit it to the article.

#hold-off

**Comment by ShannonLeavitt** [comment]
@igorpag - I found the cross-repo target. I think we're good to go. I'll commit the proposed suggestion and then merge.

**Comment by cmcclister** [comment]
Learn Build status updates of commit _[e772191](https://github.com/WillyMoselhy/cloud-adoption-framework/commits/e772191c674341c436cdbb0f2eb409880585a3be)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md](https://github.com/WillyMoselhy/cloud-adoption-framework/blob/patch-1/docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/0286ad65-6ec8-0c70-0cae-2edc1504c701/202212191918180962-1031/BuildReport?accessString=e22db110d4211f14d5b7c1fc1a7e1972f1b7e78aef8fab5417b82342a09293fd).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0286ad65-6ec8-0c70-0cae-2edc1504c701?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Pull request merged**